bin/plotting/pycbc_plot_vt_ratio

```diff
@@ -80,14 +85,27 @@ for idv in idxs:
                 np.divide(errlow2, data2)**2), ys)
     yerr_errhigh = np.multiply(np.sqrt(np.divide(errhigh1, data1)**2 +
                 np.divide(errhigh2, data2)**2), ys)
-    ax_mi.errorbar(xpos + xpos_add_dx * (colorcount - float(len(args.ifars) - 1) / 2.), ys,
+    if args.log_x:
+        xvals = np.exp(np.log(xpos) +
+                       xpos_add_dx * (colorcount -
+                                      float(len(args.ifars) - 1) / 2.))
+    else:
+        xvals = xpos + xpos_add_dx * (colorcount -
+                                      float(len(args.ifars) - 1) / 2.)
+    ax_mi.errorbar(xvals, ys,
         yerr=[yerr_errlow, yerr_errhigh], fmt='o', markersize=7, linewidth=5,
-        label='IFAR = %d yr' % ffirst_set['xvals'][idv], capsize=5, capthick=2, mec='k',
-        color=colors[colorcount % 10])
-    plt.xticks(xpos, labels, rotation='horizontal')
+        label='IFAR = %d yr' % ffirst_set['xvals'][idv], capsize=5,
+        capthick=2, mec='k', color=colors[colorcount % 10])
     colorcount += 1
 
-# get the limit of the x axes, and draw a black line in order to highlight equal comparison
+if args.log_x:
+    plt.semilogx()
+    plt.xticks(xpos, labels, rotation='horizontal')
```

Comment:
not sure why we need and else and why this xticks line turns up twice - just do it outside the 'if'.